src/main/java/core/basesyntax/report/report/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.report.report;
+
+import core.basesyntax.model.FruitOperation;
+import java.util.List;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    private static final String SEPARATOR = ",";
+    private static final String HEADER = "fruit,quantity";
+    private final StringBuilder stringBuilder;
+
+    public ReportGeneratorImpl() {
+        stringBuilder = new StringBuilder();
+        stringBuilder.append(HEADER);
+    }
+
+    @Override
+    public String getReport(List<FruitOperation> store) {
```

Comment:
```suggestion
    public String getReport() {
```
You should get data from the Storage